pysparkling/stat_counter.py

```diff
@@ -15,12 +15,9 @@
 # limitations under the License.
 #
 
+from collections import namedtuple
```

Comment:
This is the only thing that looks weird. All other files are good.

Reply:
Why do you say this looks weird? collections comes before copy? So the order is fine? The diff looks weird because actually only the __future__ is removed, but the diff states that this import statement is both removed and added... Weird indeed.

Reply:
Ok, I think I fixed it.

Reply:
It moved collections above the comment:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/2775078/106771345-24b8dd80-663f-11eb-8be6-115c1574f7a5.png">